src/core/include/openvino/core/model.hpp

```diff
@@ -114,6 +114,9 @@ class OPENVINO_API Model : public std::enable_shared_from_this<Model> {
     /// \brief Clones the original model
     std::shared_ptr<ov::Model> clone() const;
 
+    /// \brief Reference to frontend maanager
+    std::shared_ptr<void> m_femgr;
```

Comment:
Why do we have this definition in public place?

Reply:
It should be move to protected or private place.